### PR TITLE
ENG-357 amended catching 400 bad request and response error details on Error class

### PR DESCRIPTION
**Pull request by jeffgo10**

### CODE REVIEW

File: packages/entando-apimanager/modules/locales/it.js [hunk 1]
```diff
@@ -5,5 +5,6 @@ export default {
     'app.noJsonReturned': 'Non è possibile raggiungere {domain}',
     'app.permissionDenied': 'Sessione scaduta',
     'app.serviceUnavailable': 'Il server non è disponibile',
+    'app.badRequest': 'Brutta Richiesta',
```
Comment (nicpuddu) [inline packages/entando-apimanager/modules/locales/it.js:8]: it should be something more like "Richiesta Errata"

File: packages/entando-apimanager/modules/api/apiManager.js [hunk 2]
```diff
@@ -169,23 +169,28 @@ export const makeRealRequest = (request, page) => {
...
     }
     return response;
   }).catch((e) => {
-    const message = `app.${normalizeErrorMessage(e.message)}`;
-    return Promise.reject(new ErrorI18n(
-      message,
-      defaultMessages[message],
-      { domain: getDomain(store.getState()) },
-    ));
+    const promise = e.response && e.response.json ? e.response.json() : Promise.resolve();
```
Comment (nicpuddu) [inline packages/entando-apimanager/modules/api/apiManager.js:185]: has this been tested?

File: packages/entando-apimanager/modules/api/apiManager.js [hunk 3]
```diff
@@ -169,23 +169,28 @@ export const makeRealRequest = (request, page) => {
...
+    return promise.then((json) => {
+      const message = `app.${normalizeErrorMessage(e.message)}`;
+      const params = [message, defaultMessages[message], { domain: getDomain(store.getState()) }];
+      if (json) {
+        params.push(json.errors);
```
Comment (nicpuddu) [inline packages/entando-apimanager/modules/api/apiManager.js:190]: why no spread operator? in this way we don't even need the if

File: packages/entando-apimanager/modules/api/apiManager.js (hunk 2 above)
Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: yes, it falls back on the parameter being undefined when `response` prop doesn't exist. But I made some changes to guarantee that `errors` param should exist regardless of the requirements.

File: packages/entando-apimanager/modules/api/apiManager.js (hunk 3 above)
Comment (jeffgo10) [reply packages/entando-apimanager/modules/api/apiManager.js]: now I thought about it, I just removed the spread operator and instead just added the 4th parameter

**Pull request merged**